packaging/omnibus/config/projects/cloudify-osx.rb

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2015 YOUR NAME
+#
+# All Rights Reserved.
+#
+
+name "cloudify"
+maintainer "Gigaspaces"
+homepage "http://getcloudify.org/"
+
+ENV['VERSION'] || raise('VERSION environment variable not set')
+ENV['PRERELEASE'] || raise('PRERELEASE environment variable not set')
+cloudify_ver=ENV['VERSION']
+cloudify_pre=ENV['PRERELEASE']
+
+override :cacerts, version: '2017-06-07', source: { md5: '9d8c2e9a93881cdf1f2a7fc3d01a6318' }
+override :pip, version: '7.1.2', source: { md5: '3823d2343d9f3aaab21cf9c917710196' }
+override :setuptools, version: '18.5', source: { md5: '533c868f01169a3085177dffe5e768bb' }
+override :zlib, version: '1.2.8', source: { md5: '44d667c142d7cda120332623eab69f40'}
+
+# Defaults to C:/cloudify on Windows
+# and /opt/cfy on all other platforms
+install_dir "/usr/local/opt/cfy"
+
+build_version "#{cloudify_ver}-#{cloudify_pre}"
+
+# Creates required build directories
+dependency "preparation"
+dependency "cloudify-manager-blueprints"
+dependency "cloudify-manager"
+dependency "script-plugin"
+dependency "aws-plugin"
+dependency "fabric-plugin"
+dependency "openstack-plugin"
+dependency "diamond-plugin"
+dependency "vsphere-plugin"
+
+# cloudify dependencies/components
+dependency "python"
+dependency "pip"
+dependency "cloudify-cli"
+
+# Version manifest file
+dependency "version-manifest"
+
+exclude "**/.git"
+exclude "**/bundler/git"
```

Comment:
Do not duplicate files -
if osx ->
  default_root "/opt"
else ->
  default_root /usr/local/opt